jadx-gui/src/main/java/jadx/gui/plugins/script/ScriptContentPanel.java

```diff
@@ -89,15 +90,13 @@ private void initUI() {
 	}
 
 	private JPanel buildScriptActionsPanel() {
-		ActionHandler runAction = new ActionHandler(this::runScript);
-		runAction.setNameAndDesc(NLS.str("script.run"));
-		runAction.setIcon(Icons.RUN);
-		runAction.attachKeyBindingFor(scriptArea, KeyStroke.getKeyStroke(KeyEvent.VK_F8, 0));
-
-		ActionHandler saveAction = new ActionHandler(scriptArea::save);
-		saveAction.setNameAndDesc(NLS.str("script.save"));
-		saveAction.setIcon(Icons.SAVE_ALL);
-		saveAction.setKeyBinding(KeyStroke.getKeyStroke(KeyEvent.VK_S, UiUtils.ctrlButton()));
+		JadxGuiAction runAction = new JadxGuiAction(ActionModel.SCRIPT_RUN, this::runScript);
+		JadxGuiAction saveAction = new JadxGuiAction(ActionModel.SCRIPT_SAVE, scriptArea::save);
+
+		runAction.setShortcutComponent(scriptArea);
```

Comment:
```suggestion
```
This line prevents action execution on button click

Reply:
No, it's responsible for enabling mouse shortcuts. I'll fix it

Reply:
Fixed.